tests/snapshots/tests.integration.test_trace_stats.test_single_span_sampling[sampling_rule1]_tracestats.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Once we actually add that rule as a span processor, wouldn't we expect the parent trace to be sampled as well, since it has `service="test"` which meets the rule: `SamplingRule(sample_rate=1, service="test")`?